scripts/kind-e2e/README.md

```diff
@@ -80,6 +80,12 @@ as with any normal k8s cluster.
 as docker images, submariner will be redeployed on the clusters from pushed images and E2E tests will be executed.
 This mode allows the developers to test their local code fast on a very close to real world scenario setup.
 
+**NOTE**: If you only want to create the test environment without running the e2e tests, you can do it by executing 
+the following command: 
+
+```bash
+make ci e2e status=create
```

Comment:
`ci` is not needed.

Reply:
If Submariner images do not exist, the script (w/o ci) will fail.
I added both options:

- `make build package e2e status=create` (or should it be `make ci e2e status=create` ? the overhead is small, but the command is shorter) 
- and  `make e2e status=create`
    
Thoughts ? 
BTW, the `status=create` argument skips the e2e tests, but does run `test_connection`, which takes around 100  sec.  I assume that it is better to spend this time, but be sure that the deployment is configured correctly.

Reply:
right. I forgot that the `ci` thing was overseen. That's something that should be fixed (not in this patch). i.e. `e2e` should work without requiring `ci`.